be/src/exec/base_scanner.cpp

```diff
@@ -144,6 +144,14 @@ bool BaseScanner::fill_dest_tuple(const Slice& line, Tuple* dest_tuple, MemPool*
         int dest_index = ctx_idx++;
         ExprContext* ctx = _dest_expr_ctx[dest_index];
         void* value = ctx->get_value(_src_tuple_row);
+        std::string expr_error = ctx->get_error_msg();
```

Comment:
Can we do this when value == nullptr? In this way, we can avoid doing this check in most cases

Reply:
Update.

I also considered  this pointer yesterday, but I found not all UDF return value is null when it set error msg to FunctionContext,  but we could make all UDF used in data load follow 
this convention.